### PR TITLE
Re-format images

### PR DESCRIPTION
**Pull request by ndorfin**
This PR:

- introduces 2x images for the site where necessary
- resizes overly large images down to their usage needs
- compresses images
- has some misc. CSS fixes / cleanups
- Supersedes #31

### CODE REVIEW

File: source/assets/public/index.css [hunk 1]
```diff
@@ -156,7 +150,7 @@ body {
   }
 
   .breakpoint-phone .nav__room {
-    paddingtop: 4em;
+    padding-top: 4em;
```
Comment (mermop) [inline source/assets/public/index.css:153]: Good point

File: source/assets/public/index.css [hunk 2]
```diff
@@ -62,12 +62,6 @@ html {
   font-size: 52.5%;
 }
 
-*, *:after, *:before {
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
-}
```
Comment (mermop) [inline source/assets/public/index.css:69]: Is this repeated? Otherwise - why remove it?

Comment (ndorfin) [reply source/assets/public/index.css:69]: @mermop Yup, repeated a few lines along

**Comment by mermop** [comment]
Thanks @ndorfin ✨

**Pull request merged**